### PR TITLE
DOC: add "See Also: draw_idle" reference to pyplot.draw

### PR DESCRIPTION
**Pull request by hukaidong**
## PR Summary

This is a simple solution following the issue [#24578](https://github.com/matplotlib/matplotlib/issues/24578). Adding a reference `draw_idle` may reduce confusion to audiences who is visiting this page.

## PR Checklist

**Documentation and Tests**
- [ ] Has pytest style unit tests (and `pytest` passes)
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] New plotting related features are documented with examples.

**Release Notes**
- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`

### CODE REVIEW

File: lib/matplotlib/pyplot.py [hunk 1]
```diff
@@ -968,6 +968,10 @@ def draw():
 
     This is equivalent to calling ``fig.canvas.draw_idle()``, where ``fig`` is
     the current figure.
+
+    See Also
+    --------
+    .draw_idle
```
Comment (tacaswell) [inline lib/matplotlib/pyplot.py:974]:
```suggestion
    .FigureCanvasBase.draw_idle
    .FigureCanvasBase.draw
```

Doing two things:
1. include the class to help clearify that these are methods not another function
2. add `draw` to help highlight that there are other `draw` methods.

**Comment by tacaswell** [comment]
Not sure what was wrong with circle, "powercycled" to see if that fixed it.

File: lib/matplotlib/pyplot.py (hunk 1 above)
Comment (hukaidong) [reply lib/matplotlib/pyplot.py]: Merged in origin commit

**Review by tacaswell — APPROVED** [review]

**Comment by tacaswell** [comment]
I am still confused about why circle is failing.

**Comment by hukaidong** [comment]
> I am still confused about why circle is failing.

You mean the circleCI pipeline? I think it fails in every pull request but don't know why. Here's what they said

> 
> Could not find a usable config.yml, you may have revoked the CircleCI OAuth app.
> Please sign out of CircleCI and log back in with your VCS before triggering a new pipeline.

**Comment by tacaswell** [comment]
Circle is running on other PRs, would you be willing to rebase your PR onto current main (or I can do it for you, but do not want to force-push to your branch with out permission)?

**Comment by hukaidong** [comment]
> Circle is running on other PRs, would you be willing to rebase your PR onto current main (or I can do it for you, but do not want to force-push to your branch with out permission)?

rebased to latest commit but the problem seems consist

**Comment by hukaidong** [comment]
> Circle is running on other PRs, would you be willing to rebase your PR onto current main (or I can do it for you, but do not want to force-push to your branch with out permission)?

BTW you are welcomed if you want to do more test and force push on this branch. That will be much more easier

**Comment by tacaswell** [comment]
If circle on #24622 passes I will merge this PR.

**Comment by tacaswell** [comment]
See https://gitter.im/matplotlib/matplotlib?at=638d07ac4abb3f4684a40439 for discussion debugging this.

Our theory is that there is something confused on the circleCI side with @hukaidong 's account which is the source of the problem.

**Comment by tacaswell** [comment]
CI passed on #24622 so I'm going to merge this PR and close mine.

**Pull request merged**